portal-api/src/v1/users/validation/rules/email-must-be-valid-email-address.js

```diff
@@ -0,0 +1,19 @@
+const { isValidEmail } = require('../../../../utils/string');
+
+module.exports = (user, change) => {
```

Comment:
Could we add some documentation?

Not sure if this is preferred in DTFS, but I find having a function name at the top and `module.exports` at the bottom nicer  e.g

```js
const emailMustBeValid = () => { };

module.exports = emailMustBeValid;
```